packages/keyring-snap-sdk/src/rpc-handler.test.ts

```diff
@@ -1,6 +1,7 @@
-import type { Keyring } from './api';
-import type { GetAccountBalancesRequest } from './internal';
-import { KeyringRpcMethod, isKeyringRpcMethod } from './internal/rpc';
+import type { Keyring } from '@metamask/keyring-api';
+import { KeyringRpcMethod, isKeyringRpcMethod } from '@metamask/keyring-api';
+import type { GetAccountBalancesRequest } from '@metamask/keyring-internal-api';
```

Comment:
The internal structure/types for requests live on the `keyring-internal-api` since it's mostly used by the clients and the bridge for their internal communication. The public API of the `keyring-api` does not use them directly.

Reply:
No longer true, those types have been moved to public API again.